### ISSUE TITLE
ACX lottery codelab: fix next/prev link styling

### ISSUE THREAD

**Issue by chalin** [dart-lang/site-webdev#1770]
Context: https://github.com/dart-lang/site-webdev/pull/1769#discussion_r208691935

**Comment by artem-galas** [comment]
@chalin 
I've made a little research and I found that `angular.scss` didn't connected to codelab part, but I don't know how to fix this. I suppose it somewhere in build script, but I'm not sure.
 
Could you please advise me a little?

**Comment by artem-galas** [comment]
@chalin Thanks for explanation, but you forget one thing :) 
All `angular_components` steps should have `layout: components` in order to use `acx.scss` but they didn't have that.

I've fixed that, but our navigation generate empty `previous`/ `next` element if `prevpage`/`nextpage` were not set. 
here a screenshot what i mean 
http://joxi.ru/82383oJCJxepMA

Probably we should fix this as well. What do you think?

**Comment by chalin** [comment]
Yes, there was a reason for excluding the `components` layout, but let's discuss this on #1870.
